src/components/Filter/Filter.jsx

```diff
@@ -20,20 +20,25 @@ export function Filter() {
       dispatch({ type: "GPU" });
     checkIsTrue("Pixel perfect Camera for memories", "categoiries") &&
```

Comment:
typo in `categories`

Reply:
Thanks didn't see that one, just saw that my camera category was not working 😅